assets/styles/index.scss

```diff
@@ -12,6 +12,7 @@ body {
   background-color: $base;
   color: $primary;
   font-family: monospace;
+  overflow-x: hidden;
```

Comment:
Wait... does this fix the side-scrolling issue?? I imagine it has something to do with content not spreading off the side of the page

Reply:
I believe so!  overflow-x can be used on block elements (which I believe the body is?) to clip extra content, or add scroll bar, or fully display it, etc.. - so I think adding it to the body "clips" the extra so there's nothing to scroll to?

Reply:
Sounds plausible to me! Thanks